src/main/java/eu/ai4work/sws/controller/SlidingDecisionController.java

```diff
@@ -0,0 +1,30 @@
+package eu.ai4work.sws.controller;
+
+import eu.ai4work.sws.model.generic.SlidingDecisionStatus;
+import eu.ai4work.sws.model.generic.SlidingDecisionRequest;
+import eu.ai4work.sws.model.generic.SlidingDecisionResponse;
+import eu.ai4work.sws.service.SlidingDecisionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+@RestController
+@RequiredArgsConstructor
+public class SlidingDecisionController {
+    private final SlidingDecisionService slidingDecisionService;
+
+    @PostMapping("/sliding-decision")
+    public SlidingDecisionResponse processSlidingDecision(@RequestBody SlidingDecisionRequest request) {
+
+        SlidingDecisionResponse response = new SlidingDecisionResponse();
+        response.setDecisionStatus(SlidingDecisionStatus.RESPONSE.getDisplayName());
```

Comment:
I would suggest to not set it as `String` but as enum value, i.e. change the type of the attribute `decisionStatus`
```suggestion
        response.setDecisionStatus(SlidingDecisionStatus.RESPONSE);
```